templates/expo-template-bare-minimum/ios/HelloWorld.xcodeproj/project.pbxproj

```diff
@@ -140,11 +140,13 @@
 			buildPhases = (
 				08A4A3CD28434E44B6B9DE2E /* [CP] Check Pods Manifest.lock */,
 				FD10A7F022414F080027D42C /* Start Packager */,
+				353F5C274CB4DD6AD9FBEA89 /* [Expo] Configure project */,
 				13B07F871A680F5B00A75B9A /* Sources */,
 				13B07F8C1A680F5B00A75B9A /* Frameworks */,
 				13B07F8E1A680F5B00A75B9A /* Resources */,
 				00DD1BFF1BD5951E006B06BC /* Bundle React Native code and images */,
 				800E24972A6A228C8D4807E9 /* [CP] Copy Pods Resources */,
+				9021400B9B019C455B5BDDD3 /* [CP] Embed Pods Frameworks */,
```

Comment:
This is added by `pod install` and might not make sense to have in a pre-pod install template.

Reply:
Could you explain more why? The other build phases added by CocoaPods are already there 🤔 Many entries in pbxproj are modified by CocoaPods and autolinking scripts and the intention of this PR is to update it so I can just open our template as a project without any changes made by `pod install`.